comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -374,6 +375,13 @@ func (ia *inventoryagent) fetchECSFargateAgentMetadata() {
 
 func (ia *inventoryagent) fetchFleetMetadata() {
 	ia.data["config_id"] = ia.conf.GetString("config_id")
+
+	// APM host-based auto injection (SSI)
+	autoInstrumentationEnabled, err := ssi.IsAutoInstrumentationEnabled()
+	if err != nil {
+		ia.log.Warnf("could not check if APM auto-instrumentation is enabled: %s", err)
+	}
+	ia.data["feature_auto_instrumentation_enabled"] = autoInstrumentationEnabled
```

Comment:
This make every binary/project that use the inventoryagent payload depends on the SSI package.
Could you use the `Set` method of the `inventoryagent` comp instead.

Also, please update the README for `inventoryagent` to document the new field so we have a trace/doc somewhere.